src/ml_tooling/__init__.py

```diff
@@ -1,6 +1,16 @@
-# flake8: noqa
 from .baseclass import ModelData
-import pkg_resources
 
-__version__ = pkg_resources.get_distribution("ml_tooling").version
-__all__ = ["ModelData", "__version__"]
+__all__ = ["ModelData"]
```

Comment:
Maybe include the new data class here, as it would always be used.

Reply:
There are three right now, Dataset, FileDataset and SQLDataset - I like leaving them under their namespace, so it's easier to discover what types there are